README.md

```diff
@@ -1,2 +1,79 @@
-# jenkins
-Integration with Jenkins
+# Leapwork Integration
+This is Leapwork plugin for Jenkins (version 2.277.1 or later)
+
+# More Details
+Leapwork is a mighty automation testing system and now it can be used for running [smoke, functional, acceptance] tests, generating reports and a lot more in Jenkins. You can easily configure integration directly in Jenkins enjoying UI friendly configuration page with easy connection and test suites selection.
+
+# Features:
+ - Setup and test Leapwork connection in few clicks
+ - Run automated tests in your Jenkins build tasks
+ - Automatically receive test results
+ - Build status based tests results
+ - Generate a xml report file in JUnit format
+ - Write tests trace to build output log
+ - Smart UI
+ 
+# Installing
+- Use maven 
+- Command: mvn package 
+- Or simply install hpi-file from the "target" folder: Manage Jenkins -> Manage Plugins -> Advanced -> Upload Plugin -> Choose that hpi-file -> Press Upload
+
+# Update 4.0.1
```

Comment:
In release notes, keep only:
- Added support for Leapwork version 2021.1 and newer
- Plugin now uses Leapwork API v4

Reply:
Updated in branch : updatefeaturebranch